Library/Formula/fmpp.rb

```diff
@@ -0,0 +1,16 @@
+class Fmpp < Formula
+  homepage "http://fmpp.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/fmpp/fmpp/0.9.15/fmpp_0.9.15.tar.gz"
+  sha256 "b893451b5450a7f35fe680e934f6903ec8143d88959dcfca5d17467fbe4142f9"
+
+  depends_on :java
+
+  def install
+    libexec.install "lib"
+    bin.write_jar_script libexec/"lib/fmpp.jar", "fmpp", "-Dfmpp.home=#{libexec} $FMPP_OPTS $FMPP_ARGS"
+  end
+
+  test do
+    system "fmpp", "-h"
```

Comment:
It's just annoying because `fmpp` doesn't take stdin/stdout. You have to pass it a filename for the input source, and a filename for the output destination. This works from the shell, but I'm not familiar with ruby and the homebrew build environment enough to implement it properly:

    $ echo '<#assign foo="bar"/><p>${foo}</p>' > input
    $ fmpp input -o output
    *** DONE ***
    
    $ cat output
    <p>bar</p>